.github/workflows/optimum_benchmark_instinct_ci.yaml

```diff
@@ -24,7 +24,7 @@ on:
 
 jobs:
   build:
-    runs-on: [self-hosted, amd-gpu, "${{ inputs.machine_type }}"]
+    runs-on: [self-hosted, amd-gpu, mi250, "${{ inputs.machine_type }}"]
```

Comment:
Do you still need `"${{ inputs.machine_type }}" `?

Reply:
yes machine_type is for single-gpu/multi-gpu in my CI